### DISCUSSION TITLE
Control Radio via Orbitron

### DISCUSSION THREAD

**Discussion by bickbick** [AlexandreRouma/SDRPlusPlus#407] [Q&A]
Hi, is it possible to control SDRpp via Orbitron Satellite tracker?
I have edited Orbitron to start SDRpp. It asks for a DDE driver, so I point it to SDRpp.exe
Is that enough or does SDRpp need a DDE module or something?
Great program by the way..

**Comment by AlexandreRouma** [comment]
SDR++ doesn't support DDE but it supports RigCTL.
I'd suggest switching to a more modern satellite tracker like gpredict that supports RigCTL.
By the way, if you're using orbitron for NOAA and Meteor satellites, SDR++ doesn't need it, it does the doppler correction automatically in DSP.

**Reply by bickbick** [reply]
OK, thanks for quick answer.
I have gpredict all set up and connected, but it doesn't start the SDR++ even though it's 'engaged' in gpredict. Have I missed something in SDR++?
How does SDR++ know I'm tracking a NOAA satellite, does gredict communicate that?

Richard

**Reply by AlexandreRouma** [reply]
gpredict can't start SDR++ automatically. The only thing the rigctl interface can handle is tuning the VFO for doppler correction and starting/stopping a recorder or decoder module on AOS/LOS
